CMakeLists.txt

```diff
@@ -330,8 +337,15 @@ if(${HAVE_BABELTRACE})
   set(WITH_BABELTRACE ${BABELTRACE_FOUND})
   set(HAVE_BABELTRACE_BABELTRACE_H ${BABELTRACE_FOUND})
   set(HAVE_BABELTRACE_CTF_EVENTS_H ${BABELTRACE_FOUND})
+  set(HAVE_BABELTRACE_CTF_ITERATOR_H ${BABELTRACE_FOUND})
 endif(${HAVE_BABELTRACE})
 
+option(DEBUG_GATHER "C_Gather debugging is enabled" OFF)
+option(HAVE_LIBZFS "LibZFS is enabled" OFF)
+option(ENABLE_COVERAGE "Coverage is enabled" OFF)
```

Comment:
@jdurgin I wasn't aware of what ENABLE_COVERAGE did before I added this option, are we still generating coverage data from the gitbuilder tarballs?